frontends/web/src/style/variables.css

```diff
@@ -37,6 +39,9 @@
     /* background colors */
     --background: var(--color-lightgray);
     --bg-transparent-dark: rgba(0, 0, 0, 0.4);
+    --background-secondary: var(--color-white);
+    --background-dark: var(--color-softblack);
+    --background-tertiary: var(--color-gray-alt);
 
```

Comment:
One more thing: could you add a comment to `/* font colors */` and to `/* background colors */` that these are the variables that should be used, in order for dark mode to work correctly?

maybe also add comment on the section `/* colors */` of colors that should NOT be used directly, to hint to other devs which ones should be used.

Reply:
Added comments to `/* font colors */` and `*background colors*/`